sources/commands/option_external.c

```diff
@@ -49,13 +73,14 @@ int	option_external(t_control control, t_treenode *node, t_param *param)
 	*control.found = 1;
 	if (param->debug_output_level & DBG_EXTERNAL_SEARCH_FOLDERS)
 	{
-		printf("searching for command in folders:\ncurrent\n");
+		printf("searching for command in folders:\n");
 		ft_dlist_print_s(param->envvar_path_head, "\n");
```

Comment:
2. we print the whole path (debugging), that is, all the folders where we will be looking for the external executable